packages/plugins/foreman-discovery-image-service/foreman-discovery-image-service.spec

```diff
@@ -0,0 +1,36 @@
+Name: foreman-discovery-image-service
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: Metapackage with dependencies for FDI
+
+Group: Applications/System
+License: GPLv2+
+URL: https://github.com/theforeman/foreman-discovery-image
+
+Requires:	foreman-proxy
+Requires:	rubygem-smart_proxy_discovery_image
+
+%description
+Metapackage with dependencies for FDI
+
+%package tui
```

Comment:
Looks like this subpackage also needs a summary according to lint

Reply:
What does `tui` stand for?

Reply:
Text User Interface? Makes sense.